### DISCUSSION TITLE
Why is "Guide to Loris React components" deprecated?

### DISCUSSION THREAD

**Discussion by aniketg66** [aces/Loris#7422] [Q&A] [answered]
[Guide to React Components](https://github.com/aces/Loris/wiki/Guide-to-Loris-React-components) page is currently deprecated although most of the components are being used. Can anyone let me know why is the guide deprecated?

**Comment by ridz1208** [comment, marked as answer]
Hi @Anigor66, My best guess would be that it's because it has not been updated for over 3-4 years (basically since it was created). That does not mean everything on the page is deprecated, it just means there might be differences between the documentation and the code since the code has been worked on over that time period.

@christinerogers might be able to give more details as she is the one who added the deprecation comment.


If you are looking for information about a specific class or element let us know and we can find the info for you.

Best
R

**Reply by aniketg66** [reply]
Understood. Thank you!

**Comment by christinerogers** [comment]
(update) This question answered both by Rida above, and offline by email.
